smos-data/package.yaml

```diff
@@ -30,3 +30,5 @@ library:
   - validity-text
   - validity-time
   - yaml
+  - Win32
+  - unix-compat
```

Comment:
Why do you need these two for the data package??
It's not even used AFAICT

Reply:
Doublechecked and confirmed. I removed both.